### PR TITLE
dev-lang/lazarus: version bump and EAPI update

### PR DESCRIPTION
**Pull request by TheChymera**
Before I move on to adding USE flags, I wanted to bump this to the newest version and update the EAPI. @Amynka

### CODE REVIEW

**Comment by Amynka** [comment]
I would prefer if you update eapi during version bump if it is possible. Then you can remove
epatch "${FILESDIR}"/${PN}-0.9.26-fpcsrc.patch and put it into PATCHES.

**Comment by TheChymera** [comment]
@Amynka all changes applied.

File: dev-lang/lazarus/lazarus-1.6.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+src_prepare() {
+	# Use default configuration (minus stripping) unless specifically requested otherwise
+	if ! test ${PPC_CONFIG_PATH+set} ; then
+		local FPCVER=$(fpc -iV)
+		export PPC_CONFIG_PATH="${WORKDIR}"
+		sed -e 's/^FPBIN=/#&/' /usr/lib/fpc/${FPCVER}/samplecfg |
+			sh -s /usr/lib/fpc/${FPCVER} "${PPC_CONFIG_PATH}" || die
+		#sed -i -e '/^-Xs/d' "${PPC_CONFIG_PATH}"/fpc.cfg || die
+	fi
+	eapply_user
```
Comment (SoapGentoo) [inline dev-lang/lazarus/lazarus-1.6.2.ebuild]: where are the PATCHES ever applied? use `default`, and do it at the beginning

**Comment by TheChymera** [comment]
@SoapGentoo done.

**Pull request closed without merging**